### PR TITLE
Download by python

### PR DESCRIPTION
**Pull request by kuivi**
add download_data.py

more checks, should works on all platforms (i hope)

### CODE REVIEW

**Comment by kuivi** [comment]
After this PR, i can switch on the data downloading in action, if we resolve the problem of downloading WPP2022_Demographic_Indicators_Medium.zip and pend-gdis-1960-2018-disasterlocations.csv

File: download_data.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+               'population']
+    
+    for subdir in subdirs:
+        create_dir(os.path.join(base_path, subdir))
+
+    ## Download and extract other files
+    ###  -------------  coastline
+    file = 'ne_10m_coastline.zip'
+    url  = 'https://naciscdn.org/naturalearth/10m/physical/ne_10m_coastline.zip'
+    if download_file(url, file):
```
Comment (koldunovn) [inline download_data.py:106]: This looks like something we can do in a loop, given configuration in yaml with list of files and maybe some additional instructions like untaring and so on.

Comment (kuivi) [reply download_data.py]: Done.
data_sources.yml - yaml config to download data with all links subfolder info ...
[download_data.py](https://github.com/koldunovn/climsight/pull/69/files/5511f6eaceb8ff1af07fae471aeabc4bb12f25b7#diff-bc1a8ac3421ca3c3bbe1a6e4fe4a30a36a153536691319435c8b465d79b9bf73) has a loop over the all these sources from **data_sources.yml**

download_data.sh - should be probably removed from git ?

**data_climate_foresight.tar** at [swift.dkrz.de](https://swift.dkrz.de/v1/dkrz_035d8f6ff058403bb42f8302e6badfbc/clisight/data_climate_foresight.tar) should be updated (it has some trash)

Comment (kuivi) [reply download_data.py]: Oh yes, we need to figure out what to do with :
WPP2022_Demographic_Indicators_Medium.zip  - can not be downloaded most of the time (some server issues)
and Disaster Location Centroids   (manual downloading is annoying)

Comment (kuivi) [reply download_data.py]: clean up and moved data_climate_foresight data
WPP2022_Demographic_Indicators_Medium.zip is on dkrz now
pend-gdis-1960-2018-disasterlocations.csv is on dkrz now
citations added to the yml and, text files next to data files

**Review by koldunovn — APPROVED** [review]

**Pull request merged**